core/src/main/java/org/mule/runtime/core/internal/processor/strategy/StreamEmitterProcessingStrategyFactory.java

```diff
@@ -206,6 +207,13 @@ public void registerInternalSink(Publisher<CoreEvent> flux, String sinkRepresent
       activeSinksCount.incrementAndGet();
     }
 
+    @Override
+    public Publisher<CoreEvent> registerInternalFlux(Publisher<CoreEvent> flux) {
```

Comment:
This method name is not really appropriate.. It seems to be more something like configureFlux or something like this.... Actually, if this were about configuring something internal, then it shouldn't return anything